frontend/src/scenes/pipeline/destinationsLogic.tsx

```diff
@@ -37,14 +37,19 @@ interface DestinationTypeBase {
     updated_at: string
     frequency: 'realtime' | 'hourly' | 'daily'
 }
+export enum DestinationTypeKind {
+    BatchExport = 'batch_export',
+    Webhook = 'webhook',
+}
```

Comment:
lets move this to `utils.tsx`
```suggestion
export enum PipelineAppType {
    BatchExport = 'batch_export',
    Plugin = 'plugin',
}
```
Because logs are broader - destination plugins logs are the same as transformation plugins logs.

Let's use the convention:
- 'kind' - filter/transformation/destination
- 'type' - batch_export or plugin

Reply:
We found that the same applies in https://github.com/PostHog/posthog/pull/19702#discussion_r1448718181, so I'll actually do this refactor in a new PR as soon as this PR's merged